### ISSUE TITLE
[Chore]: Clean up cart.php

### ISSUE THREAD

**Issue by ZinkNotTheMetal** [ZinkNotTheMetal/tennessee-game-days#65]
### New feature you are looking for

We need to clean up cart.php so that we can modify it more easily. I know you mentioned cleaning it up so here's a tracker if you want it.

### Any additional context or links?

I sent Russ the following email on 4/1/24 @ 2:52 PM EST:

> 1. Do we have a page that the user lands on when coming back from paypal?
> 2. Are there any php files that are in TGD that you land on coming back from paypal?
> 3. Can we test this file? I want to make sure that my calls and the existing calls still work?
> 4. When creating a barcode from the indvidiual_registrations it’s ‘year-rowNumber’

Waiting for a response, when received will put it in the comment below

**Comment by ZinkNotTheMetal** [comment]
## Russ' response

1/2. I opened cart.php, and it sure looks to me like it hands the user back to completepayment.php (attached). But when I go to https://tngamedays.com/completepayment.php , I get a 500 error. I don't see any other landing page though.

3. Tell me how to test it, and I can make notes when I do. Or you can go to https://tngamedays.com/cart.php and run a test w/o me.

4. The only slight concern I would have is in length of barcode when printed. We've been using a 2 digit year, and I wouldn't want to go to 4 without having a look at the badge area and seeing if I'd have to do a slight shift in stuff. That would be OK if a 4 digit is preferable on the back end. At the risk of being short sighted, I'd hope we aren't using this same stuff in 2124. :)

**Comment by stanbery** [comment]
I've got this. Flow is:  cart.php opens with the form to be filled; it then opens itself again with the POST data, displaying it for review, providing the PayPal button; it then sends to PayPal, who then sends back to completepayment.php, **and also** hits the paypal.php when transactions happen (even if they aren't from the website).

That said, lots of refactoring and work here over the past month:

1. Moved all the bits around the cart, to make it follow a functional flow, vs PHP and HTML intermixed all over.
2. Added a config PHP file, which provides all the major variables, including turning on/off registration - this divorces it from any other scripts, like the complete and paypal.
3. Updated the complete script to execute properly (no more 500s).
4. Updated the paypal script to validate data, log the right kinds of transaction, and feed the backend database.

All of this is on the website, not in Github. Have an off-line archive, which has some magical bits we want to control.
Recommend closure.
